tests/test_adb_device.py

```diff
@@ -519,6 +519,18 @@ def test_list(self):
         self.assertEqual(expected_result, self.device.list('/dir'))
         self.assertEqual(expected_bulk_write, self.device._transport._bulk_write)
 
+    def test_list_empty_path(self):
```

Comment:
Can you please add these same tests for the async code.

Reply:
Oh, of course. I thought about it while I was writing them and then spaced! I'll get those up now, thanks for the reminder.

Reply:
Should be sorted now!